### PR TITLE
Remove augeas defaults and allow modules to specify

### PR DESCRIPTION
**Pull request by jyaworski**
Tag @igalic

### CODE REVIEW

**Comment by jyaworski** [comment]
Don't merge this yet. I'm still working on it....

**Comment by jyaworski** [comment]
Okay @igalic. Ready for review.

**Comment by jyaworski** [comment]
Actually, hold. It looks like PUPPET_VERSION isn't being honored.

**Comment by jyaworski** [comment]
We're good now.

File: moduleroot/.travis.yml [hunk 1]
```diff
@@ -5,12 +5,24 @@ sudo: false
...
     sources:
-      - augeas
+      <%- @configs['addons']['apt']['sources'].each do |x| -%>
+      - <%= x %>
+      <%- end -%>
+    <%- end -%>
+    <%- if @configs['addons']['apt'].has_key?('packages') -%>
     packages:
-      - libaugeas-dev
+      <%- @configs['addons']['apt']['packages'].each do |x| -%>
```
Comment (igalic) [inline moduleroot/.travis.yml]: ♥

**Pull request merged**